samples/snippets/snippets_mute_config.py

```diff
@@ -0,0 +1,236 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# [START securitycenter_create_mute_config]
+from google.cloud.securitycenter_v1 import Finding
+
+
+def create_mute_rule(parent_path: str, mute_config_id: str) -> None:
+    """
+    Creates a mute configuration under a given scope that will mute
+    all new finding which match the filter/ mute rule.
+    Existing findings will NOT BE muted.
+    Args:
+        parent_path: use any one of the following three options,
+                    organizations/{organization} or
+                    folders/{folder} or
+                    projects/{project}
+        mute_config_id: Set a unique id; max of 63 chars.
+    """
+
+    from google.cloud import securitycenter
+
+    client = securitycenter.SecurityCenterClient()
+
+    mute_config = securitycenter.MuteConfig()
+    mute_config.description = "Mute low-medium IAM grants excluding 'compute' "
+    # Set mute rule(s).
+    # To construct mute rules and for supported properties, see:
+    # https://cloud.google.com/security-command-center/docs/how-to-mute-findings#console_3
+    mute_config.filter = (
+        'severity="LOW" OR severity="MEDIUM" AND '
+        'category="Persistence: IAM Anomalous Grant" AND '
+        '-resource.type:"compute"'
+    )
+
+    request = securitycenter.CreateMuteConfigRequest()
+    request.parent = parent_path
+    request.mute_config_id = mute_config_id
+    request.mute_config = mute_config
+
+    mute_config = client.create_mute_config(request=request)
+    print(f"Mute Config Rule created successfully: {mute_config.name}")
+
+
+# [END securitycenter_create_mute_config]
+
+
+# [START securitycenter_delete_mute_config]
+def delete_mute_rule(mute_config_name: str) -> None:
+    """
+    Deletes a mute configuration given its resource name.
+    Note: Already muted findings are not affected when a mute config is deleted.
+    Args:
+        mute_config_name: Specify the name of the mute config to delete.
+        Its format is:
+        organizations/{organization}/muteConfigs/{config_id} or
+        folders/{folder}/muteConfigs/{config_id} or
+        projects/{project}/muteConfigs/{config_id}
+    """
+    from google.cloud import securitycenter
+
+    client = securitycenter.SecurityCenterClient()
+
+    request = securitycenter.DeleteMuteConfigRequest()
+    request.name = mute_config_name
+
+    client.delete_mute_config(request)
+    print(f"Mute Config Rule deleted successfully: {mute_config_name}")
```

Comment:
Same as above (please apply throughout this file):

Mute configuration deleted...
OR
Mute rule deleted...